sky/engine/core/script/dart_init.cc

```diff
@@ -102,6 +102,10 @@ static const char* kDartStartPausedArgs[]{
     "--pause_isolates_on_start",
 };
 
+static const char* kCompleteTimelineArgs[]{
+    "--complete-timeline",
```

Comment:
The other flags use underscore conventions here. Should this be `--complete_timeline`?